### ISSUE TITLE
Changing voxel size for inference speed

### ISSUE THREAD

**Issue by SiftingSands** [skyhehe123/SA-SSD#16]
I've been looking at ways to further increase inference speed, and my first attempt was to increase the voxel size to reduce the tensor input size to the model.

However, the only way I was able to get that to work with the pretrained network that was provided, was to decrease the "output_shape" in SpMiddleFHD. Probably not surprisingly, the model performance was incredibly poor.

I was curious if there's any configuration setting that I'm missing along these lines. Otherwise, would it be correct to assume that changing the voxel_size would necessitate retraining the model?

Thanks!

**Comment by abhigoku10** [comment]
@SiftingSands  what is the default inference speed your are able to achieve with the model ?

**Comment by SiftingSands** [comment]
I was able to get 11 fps with an RTX 3000 (mobile) GPU. I have no doubts about others claiming 25 fps on an RTX 2080ti.

**Comment by abhigoku10** [comment]
@SiftingSands can you please share your setup environment , i not getting the fps nor the accuracy metrics with the pretrained model

**Comment by SiftingSands** [comment]
CUDA 10.0
spconv 1.0
PyTorch 1.1+

**Comment by SiftingSands** [comment]
@abhigoku10 I'm not sure that's relevant to this ticket about increasing voxel size. If you have reproducibility issues, please create a separate ticket.
